website/docs/reference/dbt_project.yml.md

```diff
@@ -61,6 +61,9 @@ The following example is a list of all available configurations in the `dbt_proj
   schema: true | false
   identifier: true | false
 
+[printer_width](/reference/global-configs/print-output#printer-width): <integer>
+[log_format](/reference/global-configs/logs): <text | json | default>
```

Comment:
I think we should close this PR in favor of [this](https://github.com/dbt-labs/docs.getdbt.com/pull/4740#discussion_r1454071220) and [this](https://github.com/dbt-labs/docs.getdbt.com/issues/4709) instead.